src/Tribe/Attendees_Table.php

```diff
@@ -321,6 +321,7 @@ public function extra_tablenav( $which ) {
 			'TB_iframe' => true,
 			'width' => 410,
 			'height' => 300,
+			'parent' => 'admin.php',
```

Comment:
Basically wondering the same thing I [asked here](https://github.com/moderntribe/tribe-common/pull/101#discussion_r66971372): the submenu page is always registered against `edit.php` rather than `admin.php` - are we confident use of `admin.php` here this won't cause problems?